src/views/CompanyView/CompanyView.jsx

```diff
@@ -27,9 +30,12 @@ export function CompanyView ({ params }) {
         </Grid>
       </Grid>
       <Grid container>
-          <Grid item xs={12} sm={8} lg={12} xl={12}><ReportRuleGraph ruleOf40={ruleOf40}/></Grid>
-          <Grid item xs={12} sm={4} lg={6} xl={6}><DesciptionCard description={description} isLoading={isLoading}/></Grid>
-          <Grid item xs={12} sm={4} lg={6} xl={6}><CompanyCard financialProfile={financialProfile} isLoading={isLoading}/></Grid>
+          <Grid item xs={12} sm={8} lg={6}><ReportRuleGraph ruleOf40={ruleOf40}/></Grid>
+          <Grid item xs={12} sm={4} lg={6}><Filter setFilters={setFilters} fillFilters={false} filters={filters}/></Grid>
```

Comment:
I think we can move Filter in a different folder, in global components.
Also, we need to add Year select in all pages.

Reply:
solved comment, filter and header information to select year has been moved to global components